pymodbus/client/serial.py

```diff
@@ -226,7 +226,9 @@ def __init__(
         Setting too large a setting affects efficiency.
         """
         self._recv_interval = round((100 * self._t0), 2) + 0.01
-        self._recv_interval = self._recv_interval if self._recv_interval < 0.05 else 0.05
+        self._recv_interval = (
```

Comment:
It is still confusing to have self._recv_interval assigned 2 times, use a local variable for the first time.